Competition2018/src/main/java/competition/Practice2018Robot.java

```diff
@@ -112,7 +112,7 @@ public DeviceInfo getClimbMaster() {
 
     @Override
     public DeviceInfo getShifterSolenoid() {
-        return new DeviceInfo(0, false);
+        return new DeviceInfo(1, false);
```

Comment:
Fixing a small bug where we applied competition shifter solenoid index to both robots